ads/webediads.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {loadScript, checkData, validateDataExists} from '../src/3p';
+
+/** Created by Webedia on 07/03/16 */
+export function webediads(global, data) {
+  checkData(data, ['site', 'page', 'position', 'query']);
+  validateDataExists(data, ['site', 'page', 'position', 'query']);
```

Comment:
In particular, I'm asking because I believe `validateDataExists` would assert on empty `data-query` as in the example.